data/json/items/classes/gun.json

```diff
@@ -86,14 +86,7 @@
     "proportional": { "reload": 0.7 },
     "extend": { "flags": [ "RELOAD_ONE", "RELOAD_EJECT", "NEVER_JAMS" ] },
     "//": "Revolvers exclude the muzzle location preventing installation of suppressors",
```

Comment:
You can't normally suppress a revolver because some of the propellant gasses escape around the tiny gap between the cylinder and the barrel, not because of anything to do with the end of the barrel. They should still be able to accept muzzle brakes and Mag-na-porting.

Reply:
I didn't change or add that comment, revolvers typically don't have threaded barrels for attaching other muzzle devices either, so they still don't get a muzzle slot.